src/applications/pact-act/containers/questions/radiation/Radiation-2-3-A.jsx

```diff
@@ -75,11 +75,11 @@ const mapDispatchToProps = {
 
 Radiation23A.propTypes = {
   formResponses: PropTypes.object.isRequired,
-  setRadiation23A: PropTypes.func.isRequired,
-  viewedIntroPage: PropTypes.bool.isRequired,
   router: PropTypes.shape({
     push: PropTypes.func,
-  }),
+  }).isRequired,
```

Comment:
We had some inconsistency in `router` prop type declarations. This change is in the name of consistency.